include/mxnet/tensor_blob.h

```diff
@@ -36,8 +36,15 @@
 #include <utility>
 #include <algorithm>
 #include "./base.h"
+
 namespace mxnet {
 
+// redefine DLPack enumeration to be backward compatible.
+const int kCPU = kDLCPU;
```

Comment:
should this be constexpr? what keeps it from generating an integer in the data segment for each file compiled?

Reply:
good catch, will change to constexpr